modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -7,12 +7,7 @@
 [id="persistent-storage-csi-vsphere-top-aware_{context}"]
 = Configuring vSphere CSI Topology
 
-{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. 
-
-[NOTE]
-====
-{product-title} on vSphere does not support multiple Datacenters.
-====
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters and datacenters, thus helping to avoid a single point of failure. 
```

Comment:
Vale flags "thus" and says use "therefore" instead, but I can not find the guidance in  either style guide. 🤷‍♀️

Reply:
I checked an can't fine that "thus" is not acceptable, so I'm going to leave it as is. Thanks for your review!